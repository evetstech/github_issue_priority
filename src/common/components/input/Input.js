import React from 'react';
import './Input.scss';

const Input = ({ onChange, value }) => {
  return (
    <input className='api-input' value={value} onChange={onChange} />
  );
};

export default Input;