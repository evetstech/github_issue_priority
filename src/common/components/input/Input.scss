.api-input {
  min-height: 30px;
  height: 5%;
  min-width: 100px;
  width: 100%;
  border-radius: 4px;
}