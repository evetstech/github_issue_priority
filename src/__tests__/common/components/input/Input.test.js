import React from 'react';
import { mount } from 'enzyme';
import Input from '../../../../common/components/input/Input';
import renderer from 'react-test-renderer';

describe('input', () => {
  it('should mount', () => {
    const wrapper = mount(<Input/>);

    expect(wrapper.length).toBeTruthy();
  });

  it('renders correctly', () => {
    const tree = renderer
      .create(<Input value={'hi'}/>)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});