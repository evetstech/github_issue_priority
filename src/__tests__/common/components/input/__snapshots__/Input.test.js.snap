// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`input renders correctly 1`] = `
<input
  className="api-input"
  value="hi"
/>
`;
